frontend-mobile/lib/widgets/coffee_call.dart

```diff
@@ -52,28 +52,33 @@ class _CoffeeCallBodyState extends State<CoffeeCallBody> {
     });
   }
 
+  void _onCoffeeCupClicked() {
+    context.read<CoffeeCallCubit>().next();
+  }
+
   @override
   Widget build(BuildContext context) {
     if (userName == '') {
       return NoUsername(onUsernameSet: _setUsername);
     }
     return Column(
       children: [
-        Row(
-          children: [
-            TextButton(
-              onPressed: context.read<CoffeeCallCubit>().join,
-              child: const Text('Join'),
-            ),
-            TextButton(
-              onPressed: context.read<CoffeeCallCubit>().leave,
-              child: const Text('Leave'),
+        Padding(
+          padding: const EdgeInsets.all(20),
+          child: Hero(
+            tag: 'coffee-cup',
+            child: Material(
```

Comment:
Do we need the `Material` here? My gut feeling tells me, this should be handled by the `MaterialApp `in `main.dart`

Reply:
Jep, your right! Thats just Copy Pasta